web-app/components/pages/login/login.jsx

```diff
@@ -0,0 +1,145 @@
+import { Box, styled, Typography, Paper, Grid, TextField, Button, InputAdornment, Link } from '@mui/material';
+import React from 'react';
+import { Link as RouterLink } from 'react-router-dom';
+import MailRoundedIcon from '@mui/icons-material/MailRounded';
+import LockRoundedIcon from '@mui/icons-material/LockRounded';
+
+const StyledBox = styled(Box)(() => ({
+    width: '100dvw',
+    height: '100dvh',
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center',
+    backgroundColor: '#f4f4f4',
```

Comment:
Can we create our own custom color palette for any usage of color in future? [This documentation](https://mui.com/system/styled/#using-the-theme) can help. We can reuse any color without hardcoding it next time

Reply:
We'll talk about this later